distribution/Containerfile

```diff
@@ -1,5 +1,5 @@
 # WARNING: This file is auto-generated. Do not modify it manually.
-# Generated by: distribution/build.py
+# Generated by: distribution/build.py --unified
```

Comment:
How do we plan to stop this changing every time someone builds a different Container image locally?